platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformCachingIntegrationTest.groovy

```diff
@@ -2045,7 +2144,7 @@ class ArtifactTransformCachingIntegrationTest extends AbstractHttpDependencyReso
         """
     }
 
-    String registerFileSizer(String fileValue) {
+    String registerFileSizer(String fileValue, boolean incremental = false) {
```

Comment:
❓ There are quite a few tests that use `multiProjectWithJarSizeTransform()` and, by extension, `registerFileSizer` that now all use non-incremental transforms. I think most of those tests should be added for incremental transforms, especially `transform is applied to each file once per build`, `transform is run again and old output is removed after it failed in previous build`,  and maybe a few more.